### PR TITLE
2573: Poi list bottom action sheet title not scrolling

### PR DESCRIPTION
**Pull request by Njivaniaina2407**
### Short description

The bottom action sheet title does not scroll with the rest of the bottom action sheet content. This looks weird and blocks rare space. 
The title (In deiner Nähe) should scroll the same as the poi list items. Preferably the styling also looks like the title of a poi detail.

### Proposed changes

Put the title in the scrollable field

### Resolved issues

Fixes: #2573

---

### CODE REVIEW

File: web/src/components/PoisMobile.tsx [hunk 1]
```diff
@@ -168,7 +173,7 @@ const PoisMobile = ({
         }
       />
       <BottomActionSheet
-        title={!canGoBack ? t('listTitle') : undefined}
+        title={!canGoBack ? '' : undefined}
```
Comment (LeandraH) [inline web/src/components/PoisMobile.tsx:176]: This now chooses between two ways that don't show a title, so we can simply remove the entire title.

File: web/src/components/PoisMobile.tsx [hunk 2]
```diff
@@ -180,6 +185,7 @@ const PoisMobile = ({
           </GoBackContainer>
         )}
         <ListContainer>
+          {!canGoBack ? <ListTitle>{t('listTitle')}</ListTitle> : undefined}
```
Comment (LeandraH) [inline web/src/components/PoisMobile.tsx:188]: You can use the logical AND here as a shorthand like this: `{!canGoBack && <ListTitle>{t('listTitle')}</ListTitle>}`. But it's also fine like this, your call.

**Review by LeandraH — COMMENTED** [review]
Very nice, thank you! Just a little extra clean-up :)
Tested in Firefox

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox, well done! :tada:

**Review by LeandraH — APPROVED** [review]
Awesome, thank you!

**Pull request merged**